### ISSUE TITLE
[support] Feature Requests

### ISSUE THREAD

**Issue by rishabh3112** [sendilkumarn/webpack-scaffold-pwa#57]
Here is the list of features that would be implemented or being discussed by the plugin development team.
**Features to be implemented**
- [ ] Tests ( E2E and Unit Tests )
- [ ] Improve High level CLI design with more info of each question

**Features under implementation**
- [ ] Have 100% TS codebase ( under #63 and #59 )
- [ ] Refactor Codebase

----
Join [Slack](https://join.slack.com/t/webpack-scaffold-pwa/shared_invite/enQtNDA4NTc5MTIzMjgzLTM5NmRjYjc4ZmFjNmY0YjBiZDhiZjIxNDcwODhiOGE5ZGIyNWU4NGU1MjVlZWNmZjMwYmMwYmJjZGJjYzcwOWY) and move to #feature-discussion channel for feature recommendation and discussion

**Comment by rishabh3112** [comment]
Yes, I am looking forward to first make our base strong too @sendilkumarn.
But, I strongly suggest having a final consensus about **Under discussion** stuffs right now as I think it will enable us to design better code base structure (so that we are able to implement these stuffs without cluttering codebase)  as these stuffs will increase the scope of this plugin itself.

**Comment by sendilkumarn** [comment]
Perfect @rishabh3112 

That is the reason, I want your views and we should have a discussion and this is my brain dump for that.

**Comment by rishabh3112** [comment]
> this is my brain dump for that.

What do you mean here @sendilkumarn?

**Comment by sendilkumarn** [comment]
I mean my idea for the v1.0 release

**Comment by AjayPoshak** [comment]
I do agree with @sendilkumarn.  First we need to make our base strong and then we can easily iterate upon that to expand and add more features.
